server/app.js

```diff
@@ -1,122 +1,21 @@
-const express = require('express');
-const logger = require('morgan');
-const bodyPaser = require('body-parser');
-const userController = require('./controllers/usercontroller.js');
-const groupController = require('./controllers/groupcontroller.js');
-const messageController = require('./controllers/messagecontroller.js');
-const association = require('./associations');
-const auth = require('./auth/auth');
+import express from 'express';
+import logger from 'morgan';
+import bodyPaser from 'body-parser';
+import usersRouter from './routes/users-route';
+import recipesRouter from './routes/recipes-route';
 
-association.associate();
+import associate from './associations';
 
-const app = express();
+associate();
 
-app.use(logger('dev'));
 
+const app = express();
+app.use(logger('dev'));
 app.use(bodyPaser.json());
-
 app.use(bodyPaser.urlencoded({ extended: false }));
 
+app.use('/api/users/', usersRouter);
+app.use('/api/recipes/', recipesRouter);
 
-// registration route
-app.post('/api/user/signup', userController.validateSignup);
-app.post('/api/user/signup', userController.checkEmailUniqueness);
-app.post('/api/user/signup', userController.checkUsernameUniqueness);
-app.post('/api/user/signup', userController.signUp);
-
-// login route
-app.post('/api/user/signin', userController.login);
-
-// create group route
-app.post('/api/group', auth.verifyToken);
-app.post('/api/group', groupController.validateGroupData);
-app.post('/api/group', groupController.createGroup);
-app.post('/api/group', groupController.getGroup);
-//  deleteGroup delete a group that was created before an error
-// app.post('/api/group', groupController.deleteGroup);
-
-
-// adding group member
-app.post('/api/group/:groupid/user', auth.verifyToken);
-app.post('/api/group/:groupid/user', userController.isGroupAdmin);
-app.post('/api/group/:groupid/user', userController.isUserValid);
-app.post('/api/group/:groupid/user', groupController.alreadyAMember);
-app.post('/api/group/:groupid/user', groupController.addMember);
-
-
-// post message to a group route
```

Comment:
Are you working on a Post App? Why do you have to remove these blocks?

Reply:
No.. it is my previous project. PostIt.
I cloned the repo and remove all unwanted files.